docs/fundamentals/toc.yml

```diff
@@ -481,6 +481,8 @@ items:
         href: code-analysis/configuration-files.md
       - name: Code quality rules
         items:
+        - name: How to suppress warnings
```

Comment:
Suppressions are not specific to code quality rules, they apply to all analyzer rules. I think the new article should probably go elsewhere.